msmarco-passage-ranking/_tools/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytrec_eval
+numpy
```

Comment:
Should the presence of this file be mentioned in README? Also, please align version pinning with `dependencies` section.